src/title_screen.c

```diff
@@ -1129,89 +1217,95 @@ static u16 TitleScreen_rand(u8 taskId, u8 field)
 
 static u32 CreateBlankSprite(void)
 {
-    CreateSprite(&sUnknownEmptySprite, 0x18, 0x90, 0);
-    return IndexOfSpritePaletteTag(2);
+    CreateSprite(&sSpriteTemplate_BlankSprite, 24, 144, 0);
+    return IndexOfSpritePaletteTag(PAL_TAG_SLASH);
 }
 
-static void SetPalOnOrCreateBlankSprite(bool32 mode)
+static void SetPalOnOrCreateBlankSprite(bool32 hasCreatedBlankSprite)
 {
     u32 palIdx;
 
-    if (mode)
+    if (hasCreatedBlankSprite)
     {
-        palIdx = IndexOfSpritePaletteTag(2);
-        LoadPalette(gGraphics_TitleScreen_FireOrLeafPals, palIdx * 16 + 0x100, 0x20);
+        palIdx = IndexOfSpritePaletteTag(PAL_TAG_SLASH);
+        LoadPalette(gTitleScreen_Slash_Pal, palIdx * 16 + 0x100, 0x20);
     }
     else
         CreateBlankSprite();
 }
 
+#define sState   data[0]
+#define sTimer   data[1]
+#define sHidden  data[2]
```

Comment:
This might more accurately be `sDeactivate`.  The sprite can be visible or invisible regardless, this will trigger the sprite to enter state 2 where it will permanently sit idle until the title screen restarts.